### PR TITLE
Enter key in <length> css ref

### PR DESCRIPTION
**Pull request by digi-booster**
I just want to say that the Enter key is a more popular name than the Return key except in Norway.

> Issue number that this PR fixes (if any). For example: 'Fixes #987654321'

> What was wrong/why is this fix needed? (quick summary only)

> Anything else that could help us review it

### CODE REVIEW

File: files/en-us/web/css/length/index.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ When animated, values of the `<length>` data type are interpolated as real, floa
 
 ### Length unit comparison
 
-The following demo provides you with an input field in which you can enter a `<length>` value (e.g. `300px`, `50%`, `30vw`) to set the width of a result bar that will appear below it once you've pressed <kbd>Return</kbd>.
+The following demo provides you with an input field in which you can enter a `<length>` value (e.g. `300px`, `50%`, `30vw`) to set the width of a result bar that will appear below it once you've pressed <kbd>Enter/Return</kbd>.
```
Comment (teoli2003) [inline files/en-us/web/css/length/index.md:109]: My understanding (cf. Wikipedia) is that they are two separate keys that have the same effects:
image

So I propose:
```suggestion
The following demo provides you with an input field in which you can enter a `<length>` value (e.g. `300px`, `50%`, `30vw`) to set the width of a result bar that will appear below it once you've pressed the <kbd>Enter</kbd> or the <kbd>Return</kbd> key.
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

**Review by teoli2003 — APPROVED** [review]

**Comment by teoli2003** [comment]
Thanks!

**Comment by digi-booster** [comment]
@teoli2003 
If I see it elsewhere can I replace it as you suggest or is there no need to do so?

**Comment by teoli2003** [comment]
Sure, just do it (in another PR).

**Comment by digi-booster** [comment]
> Sure, just do it (in another PR).

(❁´◡`❁)

**Comment by teoli2003** [comment]
But don't hunt them :-)

**Pull request merged**